packages/contracts/test/quadraticFundingCollectionTests/qFundingCollectionTest.unit.ts

```diff
@@ -148,16 +89,13 @@ export const shouldBehaveLikeQuadraticVoteModule = () => {
       await ethers.provider.send("evm_mine", [currentBlockTimestamp + 750]); /* wait for round to start */
 
       //encode collect call data
-      const collectData = ethers.utils.defaultAbiCoder.encode(
-        ["address", "uint256"],
-        [_WETH.address, DEFAULT_COLLECT_PRICE],
-      );
+      const collectData = ethers.utils.defaultAbiCoder.encode(["address", "uint256"], [_WETH.address, DEFAULT_VOTE]);
 
-      await _WETH.connect(_userTwo).approve(_qVoteCollectModule.address, DEFAULT_COLLECT_PRICE);
+      await _WETH.connect(user2).approve(_qVoteCollectModule.address, DEFAULT_VOTE);
 
-      await expect(
-        _qVoteCollectModule.connect(_userTwo).processCollect(22, _userTwo.address, 1, 1, collectData),
-      ).to.emit(_votingStrategy, "Voted");
+      // TODO when fixed expect "Voted event"
```

Comment:
what's going on here?  I thought this worked earlier?

Reply:
Yes, when the funds were transfered into the module and subsequently routed to the Rounds. This 1) makes the Module the voter and 2) introduces an unintuitive money flow